### PR TITLE
Harmony: EstimateDropdown

### LINKED ISSUE #672: Harmony: EstimateDropdown
image
image
image
image

### PR DESCRIPTION
**Pull request by LamaEats**
it isn't Estimate is a DatePicker ✌️

Same returned value as original component
Simplest implementation

## PR includes

- [x] Feature

## Related issues

Resolve #672 

## QA Instructions, Screenshots, Recordings
 
image
image
image

Can be only year
image

or only single date
image

### CODE REVIEW

File: src/harmony/DatePicker/DatePicker.module.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    width: 4ch;
+}
+
+.DatePickerInlineWrapper + .DatePickerInlineWrapper,
+.DatePickerQuarter + .DatePickerQuarterAlias {
+    margin-top: var(--gap-xs);
+}
+
+.DatePickerStrictDateAdvice {
+    padding-left: calc(95px + var(--gap-s));
```
Comment (awinogradov) [inline src/harmony/DatePicker/DatePicker.module.css:95]: 95px?

**Review by awinogradov — APPROVED** [review]

File: src/harmony/DatePicker/DatePicker.module.css (hunk 1 above)
Comment (LamaEats) [reply src/harmony/DatePicker/DatePicker.module.css:95]: This offset for correct alignment by right column

Comment (awinogradov) [reply src/harmony/DatePicker/DatePicker.module.css:95]: Nobody knows. Use comments to explain magic.

File: src/harmony/DatePicker/DatePicker.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { Meta, StoryObj } from '@storybook/react';
+
+import { DatePicker, DatePickerQuarter, DatePickerStrict, DatePickerYear } from './DatePicker';
+
+const translates = {
+    year: {
+        title: 'Year',
+        trigger: 'Choose Year',
```
Comment (9teen90nine) [inline src/harmony/DatePicker/DatePicker.stories.tsx:9]: you have quarter and date start with lowcase and year in camel

**Review by 9teen90nine — APPROVED** [review]
The only thing that bothers me is that DatePicker.tsx is huge and probably should be split into parts

**Pull request merged**